index.html

```diff
@@ -12,30 +12,25 @@
 
     <script id="project-template" type="text/x-handlebars-template">
 
-    <article data-category="{{category}}" data-author="{{author}}">
-      <header>
-      <h1>{{title}}</h1>
-      <div class="byline">
-        By <address><a href="{{authorUrl}}">{{author}}</a></address>
-        published <time pubdate datetime="{{publishedOn}}">{{daysAgo}} days ago</time>
+      <div class="projectsTemp">
+        <h1>{{name}}</h1>
+        <section id="work" class="currentProjects">
```

Comment:
**TODO FOR LOST POINTS**

Please ``kabob-case`` HTML attributes as this is professional convention (though it doesn't break your code). Line 17 should be:

```
<section id="work" class="current-projects">

```